libcudacxx/CMakeLists.txt

```diff
@@ -44,43 +44,12 @@ set(LLVM_PATH "${libcudacxx_SOURCE_DIR}" CACHE STRING "" FORCE)
 
 # Configuration options.
 option(LIBCUDACXX_ENABLE_CUDA "Enable the CUDA language support." ON)
-set(_libcudacxx_enable_static_library OFF)
-if ("${CMAKE_CUDA_COMPILER_ID}" STREQUAL "NVHPC")
-  set(_libcudacxx_enable_static_library ON)
-endif ()
-option(LIBCUDACXX_ENABLE_STATIC_LIBRARY "Enable building the full C++ stdlib static library build."
```

Comment:
It was so that we could build as a standard library. We'd emit the non-header-only backends for things like `<filesystem>` and some of the various ABI compatibility bits for libstdc++. 

At this point it might be deemed unnecessary.

Reply:
Yeah, this was an option to build a trickly repackaged .a file containing libc++, libc++abi, and libunwind. It's good to go away.